docs/usermanual.md

```diff
@@ -544,3 +545,94 @@ To ensure consistent Markdown formatting run the following:
 # Use git-bash on Windows
 npm run prettier:text
 ```
+
+### Swagger
+
+In order to run Swagger, you need to execute the IoT Agent
+(https://github.com/telefonicaid/iotagent-ul/blob/master/docs/installationguide.md#usage) and then you can access to:
+
+```
+<server_host>:7896/api-docs
+```
+
+If you want to test the HTTP Protocol, two importants points:
+
+-   you should know that other services are needed (see
+    https://github.com/telefonicaid/iotagent-ul/blob/master/docs/installationguide.md#installation):
+
+    -   Mosquitto
+    -   Mongo
+    -   Rabbitmq
+    -   Orion
+
+-   you need to Provisioning a Device and Provisioning a Service Group (see this tutorial
+    https://fiware-tutorials.readthedocs.io/en/latest/iot-agent/index.html#connecting-iot-devices)
```

Comment:
Remove URL from text - use a link e.g.

```md
[tutorial](https://fiware-tutorials.readthedocs.io/en/latest/iot-agent/index.html#connecting-iot-devices)
```

Reply:
fixed in 8f66e1a31efaf583c99a9625d92709ffe5057c25